### PR TITLE
fix muting video before play (muted on the video element)

### PR DESCRIPTION
**Pull request by BrandonOCasey**
rebuild swf for change

### CODE REVIEW

**Comment by gkatsev** [comment]
No need to include the built swf (we should probably just gitignore it), we'll rebuild it when deploying.

File: src/VideoJS.as [hunk 1]
```diff
@@ -128,6 +128,10 @@ package{
                 _app.model.preload = String(loaderInfo.parameters.preload);
             }
 
+            if(loaderInfo.parameters.muted != undefined){
+                _app.model.muted = loaderInfo.parameters.muted;
```
Comment (gkatsev) [inline src/VideoJS.as]: this basically loads in the `muted` attribute value when set? I can't believe that we this was the issue, lol.

**Comment by gkatsev** [comment]
LGTM

**Comment by BrandonOCasey** [comment]
Should I just git rm the `dist` dir? or just `dist/video-js.swf`

**Comment by BrandonOCasey** [comment]
should I remove/ignore all of `dist` or just `dist/video-js.swf`?

**Comment by dmlap** [comment]
@BrandonOCasey just remove the `dist` update from this PR (`git reset HEAD dist/`, I think).

**Comment by BrandonOCasey** [comment]
guess I committed it again when I added the if check for loop, I fixed it

**Pull request merged**